replication/binlogsyncer.go

```diff
@@ -62,6 +65,15 @@ func NewBinlogSyncer(serverID uint32, flavor string) *BinlogSyncer {
 	return b
 }
 
+func (b *BinlogSyncer) LocalHostname() string {
+
+	if b.MyHost == "" {
```

Comment:
Sorry, that should have been LocalHost, not MyHost. I started with MyHost, then change my mind on the name. I just missed this when I redid the changes on my local computer. I'm surprised that the compiler did not catch this?